src/main/java/com/hydratereminder/HydrateReminderPlugin.java

```diff
@@ -26,6 +26,7 @@
 package com.hydratereminder;
 
 import com.google.inject.Provides;
+import com.hydratereminder.dictionary.HydrateWelcomeMessageDictionary;
```

Comment:
For the other dictionary class, the message retrieval method was directly imported into this class: 
`import static com.hydratereminder.dictionary.HydrateBreakMessageDictionary.getRandomHydrateBreakMessageForPersonality;`

Can we do the same for this one as well to keep things consistent?

Reply:
Yup, good catch!